### PR TITLE
Fix Trivy findings

### PR DESCRIPTION
**Pull request by fabiodmota**
## Description
Fix Trivy scan finding on DockerFile against nginx image

https://github.com/eclipse-tractusx/vas-country-risk/issues/42

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by alexsilva-CGI — APPROVED** [review]
Trivy seems ok, code changes as well

File: .github/workflows/trivy.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
         uses: aquasecurity/trivy-action@master
         with:
           # Path to Docker image
-          image-ref: "ghcr.io/catenax-ng/tx-vas-country-risk-frontend:main"
+          image-ref: "ghcr.io/catenax-ng/tx-vas-country-risk-frontend:latest"
```
Comment (SebastianBezold) [inline .github/workflows/trivy.yml:74]: I think this should stay on `:main`. The default trivy scans should scan the default branch. 
@scherersebastian what do you think about that?

File: README.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ country risk is a more general term that generally refers only to risks affectin
 ## Container images
 
 This application provides container images for demonstration purposes.
-The base image used, to build this demo application image is `node:18-alpine` and `nginxinc/nginx-unprivileged:stable-alpine`
+The base image used, to build this demo application image is `node:20-alpine` and `nginxinc/nginx-unprivileged:alpine`
```
Comment (SebastianBezold) [inline README.md:20]: I think you should introduce separate, dedicated notice for docker image files. Right now you have two sections in your top-level README about images and the one dedicated file only contains info about the frontend base image.
I'll create an issue for it describing the details. If I missed something, we can close it right away

Comment (SebastianBezold) [reply README.md:20]: Issue created: #45

File: .github/workflows/trivy.yml (hunk 1 above)
Comment (fabiodmota) [reply .github/workflows/trivy.yml:74]: done

**Comment by SebastianBezold** [comment]
[98aa894](https://github.com/eclipse-tractusx/vas-country-risk/pull/44/commits/98aa89440977942a4784f4c6a55a61c148ef9899) fixes #45

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**